CONTRIBUTING.md

```diff
@@ -63,21 +63,43 @@ When reporting an issue, details are key. Include the following:
-  * **on hold** - an idea that gained momentum but has not yet been put into a maintainer's queue to complete. Used to inform any trackers of this status
-  * **tracked** - this issue is in the JIRA backlog for the team working on snap
-  * **duplicate** - used to tag issues which are identical to other issues _OR_ which are resolved by the same fix of another issue (either case)
-  * **wontfix** - the universal sign that we won't fix this issue. This tag is important to use as we separate out the nice-to-have features from our strategic direction
+### Issue Labels
+Snap maintainers have a set of labels we use to keep up with issues. They are separated into namespaces: 
+
+* **type/** - the category of issue. All issues will have one or more
+* **reviewed/** - indicator a maintainer reviewed the issue. All issues should have one or more
+* **breaking-change/** - added to an Issue to note its merge would result in a change to existing behavior throughout the framework
+* **component/** - issues related to a particular package in the framework
+* **area/** - issues related to an overall theme and does not map to a single package
+* **effort/** - amount of work to do related to resolving or merging this code change
+
+Other indicators: 
+* **reviewed/on-hold** - an idea that gained momentum but has not yet been put into a maintainer's queue to complete. Used to inform any trackers of this status
+* **tracked** - this issue is in the JIRA backlog for the team working on snap
+* **reviewed/duplicate** - used to tag issues which are identical to other issues _OR_ which are resolved by the same fix of another issue (either case)
+* **reviewed/wont-fix** - the universal sign that we won't fix this issue. This tag is important to use as we separate out the nice-to-have features from the maintainer's agreed upon strategic direction
+* **wip-do-not-merge** - specifically for a PR to indicate it is not ready for review just yet
+
+The difference between bugs, features and enhancements can be confusing. To be extra clear, we reduced it down to two options. Here are their definitions: 
+* **type/bug** - the classic definition of missing or misbehaving code from existing functionality (this includes malfunctioning tests)
+* **type/feature-or-enhancement** - any new functionality or improvements/enhancements to existing functionality. We use one label because it's prioritized in identical ways during sprint planning
+
+For the sake of clarity, here are a few scenarios you might see play out. 
+
+As a maintainer: 
+* An issue is opened stating that Snap is not working. Upon review, the maintainer finds it is an issue with a plugin. She will label the issue with **type/wrong-repo** and open a new issue under the plugin where she tags the original issue reporter, links the original issue and labels it **bug** (which is available in plugins repositories).
```

Comment:
Probably **type/wrong-repo**  should be replaced by [**reviewed/wrong-repo](https://github.com/intelsdi-x/snap/issues?q=label%3Areviewed%2Fwrong-repo+is%3Aclosed)